### ISSUE TITLE
Positional Argument Aliases have faulty help description

### ISSUE THREAD

**Issue by Linard11** [yargs/yargs#1174]
When using positional argument aliases like `command <id|name>` and defining both positionals via `yargs.positional()` if will generate a faulty help mixing the description and [type] of both. 

**MCVE:**
```
test.js

const yargs = require('yargs');

yargs.command({
    // language=TEXT
    command: 'com <ID|NAME>',
    describe: 'The Command',
    builder: yargs => {
        yargs.positional('ID', {
            type: 'number',
            describe: 'the id',
        });
        yargs.positional('NAME', {
            type: 'string',
            describe: 'the name'
        });
    }
});

yargs.parse();
```

**Expected behavior:**
node test.js --help

test.js com <ID|NAME>

The Command

Positionals:
  ID  the id                                                 [number] [required]
  NAME  the name                                             [string] [required]

Options:
  --help     Show help                                                 [boolean]
  --version  Show version number                                       [boolean]

**Actual behavior:**
node test.js --help

test.js com <ID|NAME>

The Command

Positionals:
  ID, NAME  the name                                         [number] [required]
  ID, NAME  the name                                         [number] [required]

Options:
  --help     Show help                                                 [boolean]
  --version  Show version number                                       [boolean]

**Comment by Linard11** [comment]
Yeah, I went with <IDENTIFIER> instead, but it feels to me that this way, the positional alias are useless when you can't have multiple alias with their own description, which would seem to be almost always the case if you alias them like this.

My intention was to have two options for the function of the positional, so you can choose either one of them, with their own description and functionality. My guess was that if I define those two positionals, like in my example, with their own type, coerce, ect., the positional parameter entered by the user would be applied to both respectively. In my example case, the value `7` would be passed to the NAME positional as a string and written into argv.NAME = "7", while the same value would also be passed to the ID positional, parsed to a number (because of the type i defined) and written into argv.ID = 7.

How it is right now, I don't even know how I would define even one positional.
Do I have to do `positional('ID|NAME', { ... })` , or just just take the first name and treat the others as meaningless alias: `positional('ID', { ... })`,?

For me the alias as they are right now, are basically meaningless and only exists in the command string that the help will print out, with no functionality whatsoever, beside having the positional value being put into multiple keys in argv, which is unnecessary if in every scenario both keys will always posses the exact same value.

On a note, if the alias really don't do much but give you multiple keys in the created argv, why are they defined like this and not consistently like all other alias, with an alias key in the positional()'s opt parameter object?

**Comment by sangaman** [comment]
I also expected the "expected behvarior" in the OP here. At a minimum, you should be able to describe the two different positional argument aliases separately. What's the point of having an alias if they must be described the same?

The yargs docs provide an example of `yargs.command('get <username|email> [password]', 'fetch a user by username or email.')`. It would currently be impossible to descibr `username` and `email` separately, if you were to do:

`.describe('email', 'the registered email address')`

The output would look like:

```
Positionals:
  username, email  the registered email address
```

That's wrong for `username` and there's currently no way to describe it separately, as far as I can tell.

**Comment by astorije** [comment]
Stopping by here as I stumbled upon the same issue. I'm defining a shared set of arguments/options for a bunch of commands (but they don't all have the same options so I can't define them globally).

In the meantime I'll have to define an extra `username-or-email` to be able to describe both, but it was nice to be able to re-use the description.
